include/ceed/ceed.h

```diff
@@ -552,8 +552,8 @@ CEED_EXTERN int CeedBasisCreateHdiv(Ceed ceed, CeedElemTopology topo,
                                     const CeedScalar *div,
                                     const CeedScalar *q_ref,
                                     const CeedScalar *q_weights, CeedBasis *basis);
-CEED_EXTERN int CeedBasisCreateProjection(CeedBasis basis_to, CeedBasis basis_from, CeedBasis *basis_project);
-CEED_EXTERN int CeedBasisCreateProjectionMatrix(CeedBasis basis_to, CeedBasis basis_from, CeedScalar **interp_project);
+CEED_EXTERN int CeedBasisCreateProjection(CeedBasis basis_from, CeedBasis basis_to, CeedBasis *basis_project);
+CEED_EXTERN int CeedBasisCreateProjectionMatrix(CeedBasis basis_from, CeedBasis basis_to, CeedScalar **interp_project);
```

Comment:
One of these days we should dump the variable names in the header files since they don't actually matter.

Reply:
They can be helpful as function signatures when using a language server. Sometimes it's obvious what the argument should be from the type (`DM`, `Ceed`, etc.), but other times it's nice to know.

Reply:
But absolutely nothing is enforcing the correctness of the names in the header.

Reply:
I remember that debate. :laughing: